pages/guides/how-to-schedule-price-updates-with-gelato.mdx

```diff
@@ -0,0 +1,125 @@
+# How to Schedule Price Updates with Gelato
+
+This guide explains how to create an automated task that updates Pyth price feeds on a schedule
+
+### Why Gelato? Introduction to Gelato
+
+Gelato is web3’s decentralized backend empowering builders to create augmented smart contracts that are automated,
+gasless & off-chain aware on all major EVM-compatible blockchains. Gelato offers Web3 Functions (W3F) that connect your
+smart contracts to off-chain data & computation by running decentralized cloud functions
+
+### Gelato W3F Pyth Task
+
+The Gelato W3F Pyth Task is a Gelato task that updates Pyth price feeds on a schedule.
+The task is deployed to IPFS and can be used by anyone to schedule price updates for any Pyth price feed.
+The task is configured via a yaml file that is hosted on GitHub gist which contains
+various parameters such as the price feeds to update, the update frequency,
+price deviation threshold and can be dynamically updated at any time.
+
+## Steps
+
+#### 1. Join Gelato Web3 Functions whitelist
+
+Web3 Functions are currently in private beta.
+Get whitelisted in the Web3 Function private Beta. If you are not, reach out to
+the Gelato team [here](https://form.typeform.com/to/RrEiARiI).
```

Comment:
are currently in private beta. In order to use them, you must be added to the whitelist. Apply to the whitelist by reaching out to the Gelato team here.